distributed-commandbus-jgroups/src/main/java/org/axonframework/jgroups/commandhandling/JoinMessage.java

```diff
@@ -74,15 +79,24 @@ public int getLoadFactor() {
 
     @Override
     public void writeExternal(ObjectOutput out) throws IOException {
-        out.writeObject(address);
+
```

Comment:
Another nit, but could you drop the whitespace here.

Reply:
I will drop it.